integrators/integrators_body.hpp

```diff
@@ -92,6 +90,71 @@ SprkAsSrknDeserializer<ODE, Method, true>::ReadFromMessage(
   }
 }
 
+template<typename RightHandSideComputation>
+struct AdaptiveStepSizeIntegratorDeserializer;
```

Comment:
Invest in a comment here explaining that the `RightHandSideComputation` determines the type of the integrator when the method may be used for several kinds of integrators (this may not hold in general as we add more integrators, and is certainly not obvious).